### DISCUSSION TITLE
Water Meter Adapter for Sensus 620

### DISCUSSION THREAD

**Discussion by ps915** [jomjol/AI-on-the-edge-device#1943] [Show and tell]
### Issue

Hey folks,

I just published a highly adjustable adapter for the Sensus 620 watermeter. Works great. Happy printing.

![PXL_20230129_154642234](https://user-images.githubusercontent.com/3420935/215338943-134ae171-b3a7-4526-b20c-f16d500375e2.jpg)

https://www.printables.com/de/model/383723-ai-on-the-edge-watermeter-sensus-620-adapter

**Comment by ps915** [comment]
![image](https://user-images.githubusercontent.com/3420935/215339047-7e85f86c-05cb-435f-9732-a8f666c95cdd.png)

**Comment by ps915** [comment]
![PXL_20230129_154705302](https://user-images.githubusercontent.com/3420935/215339061-ef06f0b1-044d-478f-b9c0-bd0b886e2f4c.jpg)

**Comment by ps915** [comment]
![image](https://user-images.githubusercontent.com/3420935/215339097-53cbd056-312e-48e7-87db-832ca8b24126.png)

**Comment by ps915** [comment]
![image](https://user-images.githubusercontent.com/3420935/215339132-c5517473-4f04-47f7-882e-22083fc71295.png)

**Comment by philipgo** [comment]
Thank you! Could you maybe name/provide a link to the exact Cam-Module you use? Mine has a reset button which is slightly too wide for the case.

**Comment by codedesperate** [comment]
Yeah, mine also have the buttons on the side :(
